modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/WorkflowAllItems.path

```diff
@@ -495,6 +495,16 @@
 	<td>//div[contains(@class, 'modal-title') and contains(.,'Track Workflow')]//..//..//div[contains(@class,'modal-body')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>TRACK_WORKFLOW_DROPDOWN</td>
```

Comment:
You dont need to create this path, try to use the macro `viewKebabMenuOptionsIsAvailable`